RELEASE-NOTES.md

```diff
@@ -3,26 +3,20 @@
 > This release is dedicated to the memory of Andy Gross. Thank you and RIP.
 
 ## Overview
+This is the first full community release of Riak since Basho, Riak's original creator, ceased business operations. A lot has happened to help bring us to this point; [bet365](https://twitter.com/bet365Tech) bought Basho's assets and donated the code to the community. They kept the Basho website running, the documents site and the mailing list (after [TI Tokyo](https://www.tiot.jp/en/) had helpfully mirrored the docs in the interim). They did a huge amount to provide continuity to the community.
 
-This is the first full community release of Riak, post-Basho's
-collapse into bankruptcy. A lot has happened, in particular [bet365](https://twitter.com/bet365Tech) bought Basho's
-assets and donated the code to the community. They kept the Basho
-website running, the documents site, the mailing list (after tiot.jp
-had helpfully mirrored the docs in the interim), and have done a huge amount to
-provide continuity to the community.
-
-The development work on this release of Riak has received significant
+The development work on this release of Riak KV has received significant
 funding from [NHS Digital](https://twitter.com/NHSDigital), who depend on Riak for Spine II, and other
-critical services. Thanks also to [ESL](https://twitter.com/ErlangSolutions), [tiot.jp](https://www.tiot.jp/), and all the other
+critical services. Thanks also to [ESL](https://twitter.com/ErlangSolutions), [TI Tokyo](https://www.tiot.jp/en/), and all the other
 individuals and organisations involved.
 
-This release of Riak is based on the last known-good release of Riak,
+This release of Riak KV is based on the last known-good release of Riak KV,
```

Comment:
Is this what we're sticking with WRT naming? It always confused me, as riak_kv is an actual repo/project

Reply:
Merely trying to avoid confusion with CS, TS and the Riak family as a whole. We covered this once before in https://github.com/TI-Tokyo/riak-community-admin/blob/master/proposal/Riak%20Community%20Proposal.md#naming-conventions

Reply:
My opinion: that's Wray/McCrory error stuff that is associated with the decline of Riak, and I contend that we want to be associated with the Sheehy error.